src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+
+  &__star {
+    @include size-y-x(16px, 16px);
+
+    margin-right: 4px;
+
+    list-style-type: none;
+
+    background-image: url(../images/star-active.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &--0 > &__star,
+  &--1 :not(:first-child),
+  &--2 > &__star:nth-child(n+3),
+  &--3 > &__star:nth-child(n+4),
+  &--4 :last-child {
+    background-image: url(../images/star.svg);
+  }
```

Comment:
I think you could use an array **@for** instead of this

Reply:
good idea